pkg/kubectl/cmd/get_test.go

```diff
@@ -269,6 +269,32 @@ func TestGetObjects(t *testing.T) {
 	}
 }
 
+func TestGetObjectsWithPrefixedNamespace(t *testing.T) {
+	pods, _, _ := testData()
+
+	f, tf, codec, _ := cmdtesting.NewAPIFactory()
+	tf.Printer = &testPrinter{}
+	tf.UnstructuredClient = &fake.RESTClient{
+		APIRegistry:          api.Registry,
+		NegotiatedSerializer: unstructuredSerializer,
+		Resp:                 &http.Response{StatusCode: 200, Header: defaultHeader(), Body: objBody(codec, &pods.Items[0])},
+	}
+	tf.Namespace = "namespaces/test"
```

Comment:
Hmmm, I change this line to `tf.Namespace = "namespaces/"` and get a panic. See my comment above :)

```
--- FAIL: TestGetObjectsWithPrefixedNamespace (0.00s)
panic: Fail in goroutine after TestEdit has completed [recovered]
	panic: Fail in goroutine after TestEdit has completed

goroutine 452 [running]:
testing.tRunner.func1(0xc420385040)
	/usr/local/go/src/testing/testing.go:622 +0x29d
panic(0x19d2000, 0xc42031fca0)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
testing.(*common).Fail(0xc420385ad0)
	/usr/local/go/src/testing/testing.go:445 +0x116
testing.(*common).Errorf(0xc420385ad0, 0x1ccd96c, 0x25, 0xc4208a7b88, 0x4, 0x4)
	/usr/local/go/src/testing/testing.go:518 +0x83
k8s.io/kubernetes/pkg/kubectl/cmd.TestEdit.func5(0xc420af9e50, 0x44, 0x1)
	/home/cloud/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/edit_test.go:264 +0x203
k8s.io/kubernetes/pkg/kubectl/cmd/util.checkErr(0x2a90000, 0xc42031fc10, 0xc42092cf00)
	/home/cloud/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/util/helpers.go:178 +0x948
k8s.io/kubernetes/pkg/kubectl/cmd/util.CheckErr(0x2a90000, 0xc42031fc10)
	/home/cloud/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/util/helpers.go:119 +0x41
k8s.io/kubernetes/pkg/kubectl/cmd.NewCmdGet.func1(0xc420270480, 0xc42023b6e0, 0x2, 0x2)
	/home/cloud/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go:122 +0xad
k8s.io/kubernetes/pkg/kubectl/cmd.TestGetObjectsWithPrefixedNamespace(0xc420385040)
	/home/cloud/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/get_test.go:288 +0x455
testing.tRunner(0xc420385040, 0x1d4fdc0)
	/usr/local/go/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:697 +0x2ca
FAIL	k8s.io/kubernetes/pkg/kubectl/cmd	7.986s
make: *** [test] Error 1
```